.vscode/launch.json

```diff
@@ -0,0 +1,69 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Python: Current File",
+            "type": "python",
+            "request": "launch",
+            "program": "${file}",
+            "console": "integratedTerminal",
+            "justMyCode": false
+        },
+        {
+            "name": "Python: OTVision GUI",
+            "type": "python",
+            "request": "launch",
+            "program": "${workspaceFolder}/view.py",
+            "console": "integratedTerminal",
+            "justMyCode": false
+        },
+        {
+            "name": "Python: Detect",
+            "type": "python",
+            "request": "launch",
+            "preLaunchTask": "Build detect debug environment",
+            "postDebugTask": "Tear down debug environment",
+            "program": "${workspaceFolder}/detect.py",
+            "args": [
+                "--path",
+                ".tmp/debug/Testvideo_Cars-Cyclist_FR20_2020-01-01_00-00-00.mp4",
+                "--filetypes",
+                "mp4",
+                "--weights",
+                "yolov5s"
+            ],
+            "windows": {
+                "args": [
+                    "--path",
+                    ".tmp\\debug\\Testvideo_Cars-Cyclist_FR20_2020-01-01_00-00-00.mp4",
```

Comment:
Is the differentiation between unix and windows necessary or can python use `/` as path separator also on windows?